components/registry-facade/pkg/registry/manifest.go

```diff
@@ -94,8 +94,11 @@ type manifestHandler struct {
 
 func (mh *manifestHandler) getManifest(w http.ResponseWriter, r *http.Request) {
 	span, ctx := opentracing.StartSpanFromContext(r.Context(), "getManifest")
+	logFields := log.OWI("", "", mh.Name)
```

Comment:
@csweichel Is `mh.Name` always the workspace intance ID? Or should we better log this as “name”?

In [lines 138](https://github.com/gitpod-io/gitpod/pull/4000/files#diff-5b66d9b7307235c24d727fa706460b5f365f2c330196905a74190e762f0ccaa5L138) and 143, it has been logged as “instanceId” before …

Reply:
so far it's always the instance ID, yes :)